cs-config/cs_config/inputs.py

```diff
@@ -0,0 +1,77 @@
+from typing import Union
+
+from paramtools import Parameters
+
+
+def convert_policy_defaults(meta_params: Parameters, policy_params: Parameters):
+    """
+    Convert defaults for taxcalc's Policy class to work with C/S.
+    """
+    policy_params.array_first = False
+    policy_params.uses_extend_func = False
+    init = policy_params.dump()
+    defaults = convert_sections(init)
+    defaults = convert_data_source(defaults, meta_params.data_source)
```

Comment:
@jdebacker I think you may be a version behind since you're missing this line: https://github.com/PSLmodels/Tax-Cruncher/blob/master/cs-config/cs_config/inputs.py#L15

Reply:
Ok - just added this line.  Thanks for reviewing the changes @hdoupe!